### PR TITLE
[Azure] Fix create peering

### PR DESCRIPTION
**Pull request by J-467**
- Create peering assumed the resource vnet name by prefixing the location with the paraglider prefix
- Updating to ensure that external vnets are supported
- Contributes to #170

### CODE REVIEW

**Review by seankimkdy — APPROVED** [review]

File: pkg/azure/sdk_handler.go [hunk 1]
```diff
@@ -344,6 +344,7 @@ func (h *AzureSDKHandler) CreateVnetPeeringOneWay(ctx context.Context, vnet1Name
 			UseRemoteGateways: to.Ptr(false),
 		},
 	}
+	utils.Log.Printf("Vnet ID: %s", fmt.Sprintf(virtualNetworkResourceID, vnet2SubscriptionID, vnet2ResourceGroupName, vnet2Name))
```
Comment (smcclure20) [inline pkg/azure/sdk_handler.go:347]: Debugging thing left in?

File: pkg/azure/sdk_handler.go [hunk 2]
```diff
@@ -483,7 +484,7 @@ func (h *AzureSDKHandler) GetSecurityGroup(ctx context.Context, nsgName string)
 	return &nsgResp.SecurityGroup, nil
 }
 
-// GetParagliderVnet returns a valid paraglider vnet, a paraglider vnet is a vnet with a default subnet with the same
+// GetParagliderVnet returns a valid paraglider vnet. A paraglider vnet is a vnet with a default subnet with the same
```
Comment (smcclure20) [inline pkg/azure/sdk_handler.go:487]: It might be worth adding to this that a "paraglider vnet" is one *created* by paraglider.

**Review by smcclure20 — CHANGES_REQUESTED** [review]
Looks great! Just some minor suggestions.

File: pkg/azure/sdk_handler.go (hunk 2 above)
Comment (J-467) [reply pkg/azure/sdk_handler.go:487]: Alright

**Review by smcclure20 — APPROVED** [review]

**Pull request merged**